### DISCUSSION TITLE
[Feature] Hide album from timeline

### DISCUSSION THREAD

**Discussion by OnlyC0RE** [immich-app/immich#12259] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Since v1.113 we have automatic sync with album. It would be really helpful to be able to hide certain album from timeline. For example screenshots album, downloaded images album, whats app album, etc...

I would love to have on my timeline only photos I took with a camera.

Thanks!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5158
